python/kvikio/_lib/libkvikio.pyx

```diff
@@ -75,15 +75,15 @@ def task_size_reset(nthreads: int) -> None:
     kvikio_cxx_api.task_size_reset(nthreads)
 
 
-cdef pair[uintptr_t, size_t] _parse_buffer(buf, size):
+cdef pair[uintptr_t, size_t] _parse_buffer(buf, size, accept_host_buffer) except *:
```

Comment:
Should we try to type these?

```suggestion
cdef pair[uintptr_t, size_t] _parse_buffer(buf, size_t size, bint accept_host_buffer) except *:
```

Note: We may also want to do this elsewhere.